doc/classes/NetworkedMultiplayerCustom.xml

```diff
@@ -24,6 +24,7 @@
 			<argument index="0" name="self_peer_id" type="int" />
 			<description>
 				Initialize the peer with the given [code]peer_id[/code] (must be between 1 and 2147483647).
+				Can only be called if the connection status is [code]CONNECTION_CONNECTING[/code]. See [method set_connection_status].
```

Comment:
Missed it in the previous PR (so should be changed there too) but you should do a direct reference to the constant so it's a hyperlink and properly validated by our scripts:
```suggestion
				Can only be called if the connection status is [constant NetworkMultiplayerPeer.CONNECTION_CONNECTING]. See [method set_connection_status].
```
Same for the ones in `set_connection_status`.

Reply:
Thanks! Fixed in latest push :-)